src/SourceBuild/tarball/content/patches/runtime/0001-Attempt1.patch

```diff
@@ -0,0 +1,59 @@
+From 93de82c03ff71f897599db2824b959d020b877bd Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Thu, 30 Sep 2021 16:08:28 -0500
+Subject: [PATCH 1/2] Attempt1
```

Comment:
Please rename this patch to be meaningful and add a thorough description or link to an issue describing the underlying problems and how this patch will be resolved.

Reply:
Changed as suggested, thanks.